packages/frontend/src/contexts/SearchContext.tsx

```diff
@@ -3,17 +3,35 @@ import React, { useState } from 'react';
 export const SearchContext = React.createContext({
   query: '',
   queryHandler: (_: string) => {},
+  checkedList: [''],
+  checkedListHandler: (itemName: string) => {},
 });
 
 export const SearchContextProvider = (props: any) => {
   const [query, setQuery] = useState('');
+  const [checkedList, setcheckedList] = useState<string[]>([]);
```

Comment:
this data type (string[]) is temporary. No information about category name, will be changed in next PR